packages/ui/src/skeleton.tsx

```diff
@@ -1,17 +1,31 @@
+import React from 'react';
 import { cn } from '@hypha-platform/ui-utils';
 
 interface SkeletonProps extends React.HTMLAttributes<HTMLDivElement> {
   width?: number | string;
   height?: number | string;
+  loading?: boolean;
+  children?: React.ReactNode;
 }
 
-function Skeleton({ className, width, height, ...props }: SkeletonProps) {
+function Skeleton({
+  className,
+  width,
+  height,
+  loading = true,
+  children,
+  ...props
+}: SkeletonProps) {
+  if (!loading) {
+    return <>{children}</>;
```

Comment:
Marginal, but you should be able to write:


```suggestion
    return children;
```